dev-packages/application-package/src/generator/frontend-generator.ts

```diff
@@ -120,6 +120,11 @@ if (cluster.isMaster) {
                 console.error(error);
                 electron.app.exit(1);
             });
+            electron.app.on('quit', function() {
+                // If we forked the process for the clusters, we need to manually terminate it.
```

Comment:
I wonder is it the same when cluster forks, should we clean up for them too? Right now electron process -> master process -> worker process -> ls process. This change kills the master process. But if the worker process is still running, when the ls process won’t auto shutdown itself according the lsp.

Reply:
I think so. I had the same feeling. But I would like to tackle that as part of #815. I plan to proceed with smaller changes and verify them one by one on each platform with the bundled electron application.

Reply:
My point is that probably nothing should be done about ls servers, only workers should be killed, and ls processes will exit according to the lsp.

Reply:
OK. We'll see how that works in practice, especially on Windows in the context of #815.